### PR TITLE
Add further text to the Server Name Indication section

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

File: draft-ietf-uta-tls13-iot-profile.md [hunk 1]
```diff
@@ -138,13 +138,19 @@ The discussion in Section 12 of {{!RFC7925}} is applicable with one exception:
...
-still at an experimental state.
+# Server Name Indication
+
+This specification mandates the implementation of the Server Name Indication (SNI)
+extension. Where privacy requirements require it, the Encrypted Client Hello
+extension {{?I-D.ietf-tls-esni}} prevents an on-path attacker to determine the domain
+name the client is trying to connect to. 
+
+Note: To avoid leaking DNS lookups from network inspection altogether further 
+protocols are needed, including DoH [RFC8484] and DPRIVE [RFC7858] [RFC8094].
```
Comment (thomas-fossati) [inline draft-ietf-uta-tls13-iot-profile.md:149]:
```suggestion
protocols are needed, including DoH {{?RFC8484}} and DPRIVE {{?RFC7858}} {{?RFC8094}}.
```

File: draft-ietf-uta-tls13-iot-profile.md [hunk 2]
```diff
@@ -138,13 +138,19 @@ The discussion in Section 12 of {{!RFC7925}} is applicable with one exception:
...
-# Server Name Indication (SNI)
-
-This specification mandates the implementation of the SNI extension. Where
-privacy requirements require it, the encrypted SNI extension
-{{?I-D.ietf-tls-esni}} prevents an on-path attacker to determine the domain
-name the client is trying to connect to. Note, however, that the extension is
-still at an experimental state.
+# Server Name Indication
+
+This specification mandates the implementation of the Server Name Indication (SNI)
```
Comment (thomas-fossati) [inline draft-ietf-uta-tls13-iot-profile.md:143]: should we use more explicit BCP14-enhanced verbs instead of just "mandates"?

Comment (hannestschofenig) [reply draft-ietf-uta-tls13-iot-profile.md:143]: In an earlier section we already say "MUST implement" and so I didn't want to repeat it again

Comment (thomas-fossati) [reply draft-ietf-uta-tls13-iot-profile.md:143]: 👍

File: draft-ietf-uta-tls13-iot-profile.md [hunk 3]
```diff
@@ -138,13 +138,19 @@ The discussion in Section 12 of {{!RFC7925}} is applicable with one exception:
...
+extension. Where privacy requirements require it, the Encrypted Client Hello
+extension {{?I-D.ietf-tls-esni}} prevents an on-path attacker to determine the domain
+name the client is trying to connect to. 
+
+Note: To avoid leaking DNS lookups from network inspection altogether further 
+protocols are needed, including DoH {{?RFC8484}} and DPRIVE {{?RFC7858}} {{?RFC8094}}.
+Since the Encrypted Client Hello extension requires use of Hybrid Public Key 
+Encryption (HPKE) {{I-D.irtf-cfrg-hpke}} and additional protocols require
```
Comment (thomas-fossati) [inline draft-ietf-uta-tls13-iot-profile.md:151]:
```suggestion
Encryption (HPKE) {{?I-D.irtf-cfrg-hpke}} and additional protocols require 
```

**Pull request merged**